include/inexor/vulkan-renderer/application.hpp

```diff
@@ -84,9 +85,9 @@ class Application : public VulkanRenderer {
 
     /// @brief Call camera's process_mouse_scroll method.
     /// @param window The window that received the event.
-    /// @param x_offset The change of x-offset of the mouse wheel.
-    /// @param y_offset The change of y-offset of the mouse wheel.
-    void mouse_scroll_callback(GLFWwindow *window, double x_offset, double y_offset);
+    /// @param xoffset The change of x-offset of the mouse wheel.
+    /// @param yoffset The change of y-offset of the mouse wheel.
+    void mouse_scroll_callback(GLFWwindow *window, double xoffset, double yoffset);
```

Comment:
Looks like the rebase is still messed up. These changes should not be here as commits, right?